inc/Project.php

```diff
@@ -132,7 +110,7 @@ class Project {
 	 *
 	 * @param \GP_Project $project GlotPress project.
 	 */
-	public function __construct( $project ) {
+	public function __construct( \GP_Project $project ) {
```

Comment:
It is already imported.

```suggestion
	public function __construct( GP_Project $project ) {
```